docs/getting-started/quickstart.md

```diff
@@ -174,10 +170,9 @@ prefect work-pool create my-managed-pool --type prefect:managed
 You should see a message in the CLI that your work pool was created.
 Feel free to check out your new work pool on the **Work Pools** page in the UI.
```

Comment:
Is this line helpful? I'd vote to remove it as it pulls the user away from the main focus.

Reply:
Good question. I think we want to get the user checking out the UI at some point, but maybe not until they run the deployment.

Reply:
You could do it all in Python code + CLI. The UI is a core part of the product that I think we want folks interacting with. That said, I hear you on bouncing back and forth being distracting.